packages/main/src/MenuItem.ts

```diff
@@ -158,6 +158,16 @@ class MenuItem extends ListItem {
 	@slot({ "default": true, type: HTMLElement, invalidateOnChildChange: true })
 	items!: Array<MenuItem>;
 
+	/**
+	 * Defines the components that should be displayed at the end of the menu item.
+	 *
+	 * **Note:** It is highly recommended to slot only components of type `ui5-button` (icon-only),
```

Comment:
Not quite sure about this note. Maybe we should only keep to what is recommended and what is the intention of providing this slot.

Reply:
Done